quinn-proto/src/connection/mod.rs

```diff
@@ -1298,10 +1298,7 @@ where
             }
             let last_ack_eliciting = match self.spaces[space].time_of_last_ack_eliciting_packet {
                 Some(time) => time,
-                None => {
-                    debug_assert!(!self.peer_completed_address_validation());
-                    now
-                }
+                None => continue,
```

Comment:
If you're going to revert it anyway, should we just drop the first commit?

Reply:
The first commit represents the absolute minimal fix to get correct behavior, while the later fix relies on stronger inferences about the behavior of the state machine. Given how subtle the logic is, I'm inclined to separate those out to make it easier to look back, and perhaps amend, our approach without risking a regression, particularly while https://github.com/quicwg/base-drafts/issues/4846 is still open.

Reply:
Okay.